docs/ballooning.md

```diff
@@ -40,6 +40,15 @@ The device can be configured with the following options:
   the virtio balloon statistics and otherwise represents the interval of time in
   seconds at which the balloon statistics are updated.
 
+The device has two optional features which can be enabled with the following
+options:
+
+- `free_page_reporting`: A mechanism for the guest to continually report ranges
+  of memory which the guest is not using and can be reclaimed. Read more here
+- `free_page_hinting`: A dev-preview feature which is a different mechanism to
```

Comment:
Maybe format it like:
```md
- (DEV PREVIEW) 'free_page_hinting': A mechanism to relaim ...
```
This way it is more visiable that it is not GA. (and maybe include link to the `../docs/RELEASE_POLICY.md#developer-preview-features`)